client/app/account/api/editAccountApi.ts

```diff
@@ -0,0 +1,11 @@
+import apiClient from '@/_api/base'
+import { UserType } from '@/_types'
+import { AccountSchema } from '../schema'
+
+export const editAccountApi = async (
```

Comment:
It's up to you but because the function name in server is update, I think it's better to define `update` instead of `edit` in client side!

Reply:
I actually considered between `update` and `edit`, but I chose `edit` for shorter names.
I renamed them to match the server side.
2c83d9aeb259db6607a5a82957c69a6df321411d